docker-compose-build.yml

```diff
@@ -53,3 +53,9 @@ services:
     build:
       context: .
       dockerfile: docker_build/cli/Dockerfile
+
+  api:
+    image: ${DOCKER_REGISTRY}cli:${DOCKER_TAG}
+    build:
+      context: .
+      dockerfile: docker_build/api/Dockerfile
```

Comment:
Regarding the prod build stuff, I want to mention three things:
- Be careful, the image name contains `cli`
- Adding it here is not enough, you will also have to specify it here to push it to the container registry https://github.com/MaskAnyone/MaskAnyone/blob/main/.github/workflows/build_push_docker.yml#L33
- Generally I'd suggest only adding the prod build stuff here once you feel that it is ready to share, during development you can just skip it alltogether

Reply:
Thanks a lot, good catch with the image name! 
Also thanks for letting me know about the .github workflow file, I adjusted it there as well. This was exactly why I wanted to get your feedback, to know if anything is missing for the build steps! Thanks :)